frontend/src/lib/k8s/namespace.ts

```diff
@@ -13,6 +13,18 @@ class Namespace extends makeKubeObject<KubeNamespace>('namespace') {
   get status() {
     return this.jsonData!.status;
   }
+
+  static isValidNamespaceFormat(namespace: string) {
+    // We allow empty strings just because that's the default value in our case.
+    if (!namespace) {
```

Comment:
Can you please add a unit test for this function?

Reply:
added a namespace.test.ts for unit testing but I am hitting a blocker with this error

![image](https://github.com/headlamp-k8s/headlamp/assets/78232183/9fe783ee-c6c3-4b7d-8438-2b76d07f7b77)


The error says the makeKubeObject is not a function, yet it is one and the import for it should be fine on the normal namespace.ts, what is even more weird is that I cannot seem to find how or why its using the Role extends as I do not see anything that Namespace class extends directly off of any Role usage? other than searching project wide for Role or ClusterRole extensions?